applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.hpp

```diff
@@ -132,6 +132,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) StressStrainUtilities
         KRATOS_CATCH("")
     }
 
+    static Matrix CalculateGreenLagrangeStrainTensor(const Matrix& rDeformationGradient);
```

Comment:
Here also all other functions can be split, such that we do not need a .hpp but a cleaner .h and .cpp files. I'd propose to do that in a separate PR.

Reply:
Yes please